packages/dapp-connector/src/translators/reactjs/useDappList.ts

```diff
@@ -0,0 +1,9 @@
+import {useQuery, UseQueryOptions} from 'react-query'
+import {dappConnectorApiGetDappList, DappListResponse} from '../../adapters/api'
+
+export const useDappList = (options?: UseQueryOptions<DappListResponse, Error, DappListResponse, 'dappList'>) => {
+  return useQuery('dappList', {
+    ...options,
+    queryFn: dappConnectorApiGetDappList(),
```

Comment:
make it receive the context and consume from the context (so we can inject anything later and mock it easily),